tsconfig.json

```diff
@@ -14,12 +14,14 @@
     "noUnusedParameters": true,
     "outDir": "dist",
     "sourceMap": true,
-    "suppressImplicitAnyIndexErrors": true
+    "suppressImplicitAnyIndexErrors": true,
+    "allowSyntheticDefaultImports": true
```

Comment:
Once disabled, many of the used packages will throw an error while type checking because it has no default export:

- jest-mock
- jest-resolve
- jest-runtime
- events
- fs
- jest-haste-map